pkg/common/series.go

```diff
@@ -0,0 +1,60 @@
+package common
+
+// NOTE: our series only have one value for each timestamp, because multiple values can be expand to single value
+// and I am not sure how InfluxDB and Druid implement multiple values @czheo
+//
+// A multiple value series
+// cpu.0 system=ubuntu, arch=amd64, usage=0.062, idle=0.034, 1412312312
+// Expand to single value series
+// cpu.0.usage system=ubuntu, arch=amd64, 0.062, 1412312312
+// cpu.0.idle  system=ubuntu, arch=amd64, 0.034, 1412312312
+
+// Series is a time series
+type Series struct {
+	// TODO: string or []byte
+	Name      string            `json:"name"`
+	TagKeys   []string          `json:"-"`
+	TagValues []string          `json:"-"`
+	Tags      map[string]string `json:"tag"`
```

Comment:
I have wrote some notes in fe8f8de310d26fb77fa19f566f111a4c9a5a6439  The two array first come from influx-comparison. The advantage is the order of tags is kept, when you loop through a map in go, the order is random, which may add some slight overhead when doing test like [this](https://github.com/xephonhq/xephon-b/blob/feature/data-generation/pkg/simulator/machine_test.go#L39) (this is not a good test indeed ...)
````go
assert.Equal("cpu_core", allSeries[0].TagKeys[1])
````
Also arrays don't need explicit initialization like `map`
The down side of using two arrays is serialization and duplication . 

I think I can go with map, since serialization is more often than looking up map values in insert order. Just need to modify some tests.